### PR TITLE
sdcm/cluster: deb: add new sign key

### PR DESCRIPTION
**Pull request by amoskong**
We have a new key to sign deb packages, and the old releases are used in
upgrade testing. So let's reserve the old keys.

Signed-off-by: Amos Kong <amos@scylladb.com>

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -1763,6 +1763,7 @@ def install_scylla(self, scylla_repo):
                     apt-get install software-properties-common -y
                     apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6B2BFD3660EF3F5B
                     apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 17723034C56D4B19
```
Comment (roydahan) [inline sdcm/cluster.py]: @amoskong, why did you leave the old keys as well?

File: sdcm/cluster.py [hunk 2]
```diff
@@ -1794,6 +1795,7 @@ def install_scylla(self, scylla_repo):
                     apt-get install gnupg1-curl dirmngr -y
                     apt-key adv --fetch-keys https://download.opensuse.org/repositories/home:/scylladb:/scylla-3rdparty-stretch/Debian_9.0/Release.key
                     apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 17723034C56D4B19
```
Comment (roydahan) [inline sdcm/cluster.py]: Let’s remove the old key

File: sdcm/cluster.py (hunk 1 above)
Comment (amoskong) [reply sdcm/cluster.py]:
```
We have a new key to sign deb packages, and the old releases are used in
upgrade testing. So let's reserve the old keys.
```

File: sdcm/cluster.py (hunk 2 above)
Comment (amoskong) [reply sdcm/cluster.py]: If we remove the old key, upgrade / rollback will fail.

Comment (roydahan) [reply sdcm/cluster.py]: Ok, didn’t think about it.
Thanks.

**Comment by roydahan** [comment]
@bentsi / @fruch please backport

**Comment by bentsi** [comment]
backported

**Pull request merged**